doc/user/content/manage/monitor/alerting.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Alerting"
+description: "Alerting thresholds to use for monitoring."
+menu:
+  main:
+    parent: "monitor"
+---
+
+After setting up a monitoring tool, it is important to configure alert rules. Alert rules send a notification when a metric surpasses a threshold. This will help you prevent operational incidents.
+
+This page describes which metrics and thresholds to build as a starting point. For more details on how to set up alert rules in Datadog or Grafana, refer to:
+
+ * [Datadog monitors](https://docs.datadoghq.com/monitors/)
+ * [Grafana alerts](https://grafana.com/docs/grafana/latest/alerting/fundamentals/)
+
+## Thresholds
+
+Alert rules tend to have two threshold levels, and we are going to define them as follows:
+ * **Warning:** represents a call to attention to a symptom with high chances to develop into an issue.
+ * **Alert:** represents an active issue that requires immediate action.
+
+For each threshold level, use the following table as a guide to set up your own alert rules:
+
+Metric | Warning | Alert | Description
+-- | -- | -- | --
+CPU | 85% | 100% | Average CPU usage for a cluster in the last *60 minutes*.
+Memory | 80% | 90% | Average memory usage for a cluster in the last *30 minutes*.
+Status | - | On Change | Source status change in the last *5 minutes*.
+Distribution | Time spent by worker > Avg * 1.5 | Time spent by worker > Avg * 2 | Average [work distribution](https://materialize.com/docs/manage/troubleshooting/#is-work-distributed-equally-across-workers) in the last *60 minutes*.
+Freshness | > 5s | > 1m | Average [lag behind an input](https://materialize.com/docs/sql/system-catalog/mz_internal/#mz_materialization_lag) in the last *30 minutes*.
+
+### Custom Thresholds
+
+For the following table, replace the two variables, _X_ and _Y_, by your organization and use case:
+
+Metric | Warning | Alert | Description
+-- | -- | -- | --
+Latency | p99 > X | p99 > Y | Latency percentile from the activity log in the last *30 minutes*. Where X and Y are the expected latencies in milliseconds.
```

Comment:
What's the query powering this? Should we instruct users to monitor a filtered subset of query patterns (or target views) they care about?

Reply:
> What's the query powering this? 

@chuck-alt-delete made this query https://materializeinc.slack.com/archives/C0638BNE256/p1700027081349059?thread_ts=1700027001.256819&cid=C0638BNE256 that we could document. As an alternative, I can switch to the avg, max, min query. Anyways, I will research what's the most common insight for these cases.

> Should we instruct users to monitor a filtered subset of query patterns (or target views) they care about?

We should. I suggested a customer filter out `mz_introspection` and `mz_system`, and we could also suggest filtering `SELECT` statements. I feel this is more a part of the monitoring guides than alerting. If you agree, I will create an extra PR for a base query for this and cover any other missing queries not yet documented.

Reply:
Good point. I do feel like like some filtering is necessary in order to set a useful alert threshold. Note in the linked query the common filter `where mpsh.redacted_sql ilike '%[view name]`. You might expect certain queries to take longer than others and wouldn't want to alert on false negatives using a universal latency ceiling.
>I feel this is more a part of the monitoring guides than alerting

Reply:
After researching and looking at our examples, I concluded to use the `avg` latency. We already have customers using it in production, and it is simpler to visualize.

Reply:
@ggnall after the discussion in Slack about how to detect `SELECT` statements I opted to avoid using filters until we have a more robust approach.